srcpkgs/sowm/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'sowm'
+pkgname=sowm
+version=1.0
+revision=1
+build_style=gnu-makefile
+makedepends="libX11-devel"
+depends="libX11"
+short_desc="Itsy bitsy floating window manager "
+maintainer="shizonic <realtiaz@gmail.com>"
+license="MIT"
+homepage="https://github.com/dylanaraps/sowm"
+distfiles="https://github.com/dylanaraps/sowm/archive/${version}.tar.gz"
+checksum=60f3deaf2663fd35da3ac305143350f4b28dd41cea5f4ffc0fd7cf7f1e268098
+
+do_install() {
+	make PREFIX=/usr DESTDIR="${DESTDIR}" install
```

Comment:
it's enough to just vlicense in post_install, as your make cmd matches the default by xbps...